lxd/device/gpu_sriov.go

```diff
@@ -85,6 +85,12 @@ func (d *gpuSRIOV) Start() (*deviceConfig.RunConfig, error) {
 	runConf := deviceConfig.RunConfig{}
 	saveData := make(map[string]string)
 
+	// Make sure that vfio-pci is loaded.
```

Comment:
We should prob move this below the lock to avoid unnecessarily calling this concurrently.